.pullapprove.yml

```diff
@@ -13,6 +13,10 @@ group_defaults:
     ignored: true
   teams:
     - dsde-pipelines-developers
+  conditions:
+    files:
+      exclude:
+        - ".clio-version" # So Jenkins can auto-bump the version
```

Comment:
In general I think end-of-line comments make code harder to read and edit. In this case I'd move this comment up, maybe before the `conditions:` line.